classes/core/PKPPageRouter.php

```diff
@@ -31,8 +31,8 @@ class PKPPageRouter extends PKPRouter
     /** @var array pages that don't need an installed system to be displayed */
     public $_installationPages = ['install', 'help', 'header', 'sidebar'];
 
-    public const ROUTER_DEFAULT_PAGE = './pages/index/index.php';
-    public const ROUTER_DEFAULT_OP = 'index';
+    public const ROUTER_DEFAULT_PAGE = './pages/' . Application::SITE_CONTEXT_PATH . '/index.php';
```

Comment:
I think the `SITE_CONTEXT_PATH` constant is at risk of being used for too many different things. At the least, the use of URLs and file paths shouldn't be mixed, and in cases like this I sometimes find that repeating the string is more readable/maintainable than burying it into a pile of pseudo-globals in `PKPApplication`.

If the purpose of the constant is to reserve the `index` context name, then it shouldn't be mixed with the default op (below) or the file path (this line).

Reply:
Yeah, my intention was to reserve the "index", if someone decides to rename it to something else, it would be an easier task.

The default op was an overlook (the other `index` usages were left untouched) and the default page a mistake (my automatic pilot told me it was a virtual path and not a physical one), I'll update both :)